tests/system/action/motion/test_update.py

```diff
@@ -277,6 +277,7 @@ def test_update_workflow_id(self) -> None:
                 },
             }
         )
+        sleep(1)
```

Comment:
Remove this again. I suppose the error stems from the fact that `created` is set to the current timestamp and then directly afterwards it is tried to assert that `created` < `workflow_timestamp`. Just replace the `<` with a `<=` and everything should work fine again.

Reply:
Done.